compose/types/chart_test.go

```diff
@@ -0,0 +1,158 @@
+package types
+
+import (
+	"encoding/json"
+	"github.com/cortezaproject/corteza-server/pkg/locale"
+	"github.com/stretchr/testify/require"
+	"reflect"
+	"testing"
+)
+
+func TestChart_decodeTranslations(t *testing.T) {
+	cc := []struct {
+		name string
+		base *ChartConfigReport
+		ccr  *ChartConfigReport
+		tt   locale.ResourceTranslationIndex
+	}{
```

Comment:
Should we also use the stringified JSON payload as you've used below? Would make sense to me